client/src/containers/Profile.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Redirect } from 'react-router';
+import { Auth } from '../Auth/Auth.js';
+
+const auth = new Auth();
+
+import { connect } from 'react-redux';
+import { fetchProducts } from '../actions/productActions.jsx';
+import { changeVideo, changeBroadcastMessage, selectFeaturedProduct } from '../actions/broadcastActions.jsx';
+
+class BroadcastView extends React.Component {
```

Comment:
Is the class name appropriate?

Reply:
No, Sorry. They were just boilerplates and I forgot to convert the names. Should be fixed now.